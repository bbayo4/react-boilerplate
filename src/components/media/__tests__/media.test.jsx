import React from 'react';
import Enzyme from 'enzyme';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

import Media from '../media';

describe('The Media Component', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<Media />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});