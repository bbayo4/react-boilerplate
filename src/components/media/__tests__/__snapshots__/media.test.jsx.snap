// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Media Component should render correctly 1`] = `
<div
  className="media-wrapper"
>
  <audio
    controls={true}
  >
    <source
      src="test-file-stub"
      type="audio/mpeg"
    />
  </audio>
</div>
`;
